src/components/Expando.tsx

```diff
@@ -66,7 +75,7 @@ export default function Expando({ title, open, onExpand, height, children, ...re
         <Rule />
         <TitleRow>
           <TitleHeader>
-            <InfoIcon style={{ marginRight: '5px' }} color="secondary" />
+            {hideIcon === true ? null : <InfoIcon style={{ marginRight: '5px' }} color="secondary" />}
```

Comment:
I think this icon should only ever be passed as a part of the title.
Can `hideIcon` and `InfoIcon` be removed, and `InfoIcon` just added explicitly to those use cases?